internal/scanner/scanner.go

```diff
@@ -142,19 +142,22 @@ func (s *Scanner) ScanNamespace(ctx context.Context, nsName, runUID string) erro
 	}
 	policies, err := s.policiesClient.GetPoliciesForANamespace(ctx, nsName)
 	if err != nil {
+		log.Error().Err(err).Str("namespace", nsName).Msg("failed to obtain auditable policies")
 		return err
 	}
+
 	log.Info().
 		Str("namespace", nsName).
 		Dict("dict", zerolog.Dict().
 			Int("policies-to-evaluate", policies.PolicyNum).
-			Int("policies-skipped", policies.SkippedNum),
+			Int("policies-skipped", policies.SkippedNum).
```

Comment:
we need to do the same for https://github.com/kubewarden/audit-scanner/blob/c841bc1d6c438db00cbf6317708a04bc8b227dca/internal/scanner/scanner.go#L255

Reply:
True! done in https://github.com/kubewarden/audit-scanner/pull/312/commits/8d4afbbe0e5249ee2707701858692033fa629e29.